eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -165,6 +170,47 @@ stages:
                           PRLabels: "auto-merge"
                           CloseAfterOpenForTesting: '${{ parameters.TestPipeline }}'
 
+                      - task: PowerShell@2
+                        displayName: Get PR owners from CODEOWNERS
+                        inputs:
+                          pwsh: true
+                          filePath: $(Build.SourcesDirectory)/eng/common/scripts/get-codeowners.ps1
+                          arguments: >
+                            -TargetDirectory "/sdk/${{ parameters.ServiceDirectory }}/"
+                            -RootDirectory "$(Build.SourcesDirectory)"
+                            -VsoVariable "PrOwners"
+
+                      - pwsh: |
+                          $owners = @()
+                          foreach($owner in $env:PrOwners -split ',') {
+                            # Exclude groups. GitHub does not notify teams
+                            # that have different owners (e.g. alerting
+                            # @Azure/example-team when opening a PR in
+                            # Microsoft/vcpkg)
+                            if ($owner.indexOf('/') -ne -1) {
```

Comment:
As potential follow-up here if we find we do this in other places is to add an option to get-codeowners.ps1 that excludes teams.

Reply:
While the names might not resolve is it actually necessary to exclude? If not perhaps we just simply things here and include all the owners.

Reply:
Excluding groups which won't resolve is done in the interest of keeping the PR text clean of text that won't generate meaningful alerts. These can be kept in without creating errors.